contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -791,8 +791,8 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
         require(
             _transaction.blockNumber        == _txChainElement.blockNumber
             && _transaction.timestamp       == _txChainElement.timestamp
-            && _transaction.entrypoint      == resolve("OVM_DecompressionPrecompileAddress")
-            && _transaction.gasLimit        == gasLimit
+            && _transaction.entrypoint      == 0x4200000000000000000000000000000000000005
+            && _transaction.gasLimit        == 7000000
```

Comment:
This was temporary, need to revert the change.

Reply:
At one point this require was split up into many requires so that its easier to tell what the problem is, placing it all in a single require to save gas?

Reply:
Hmmm that might've been a different `require` statement, this one was always in a single statement.